anthos-bm-utils/abm-nettest/README.md

```diff
@@ -0,0 +1,105 @@
+## What is this?
+
+ABM nettest enables users/support members to find any connectivity issues
+among k8s pods/nodes/svcs/external targets (except for connections from external
+targets to pods/nodes/svcs). This playbook will help to quickly spot any
+connectivity issues among k8s components in the cluster of interest.
+
+## Quickstart
+
+### Run Nettest
+
+Run the nettest by running the following commands.
+
+For non-RHEL OS:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/anthos-samples/main/anthos-bm-utils/abm-nettest/nettest.yaml
+```
+
+For RHEL OS:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/anthos-samples/main/anthos-bm-utils/abm-nettest/nettest_rhel.yaml
+```
+
+NOTE: private registry users needs to download the manifest and modify all image names respectively.
+
+### Get test results
+
+Run the following command to see the nettest result:
+
+```
+kubectl -n nettest logs nettest nettest
+```
+
+While the nettest is running, you will see messages like the following:
+
+```
+I0413 03:33:04.879141       1 collectorui.go:130] Listening on ":8999"
+I0413 03:33:04.879258       1 prometheus.go:172] Running prometheus controller
+E0413 03:33:04.879628       1 prometheus.go:178] Prometheus controller: failed to retries probers: Get "http://127.0.0.1:9090/api/v1/targets": dial tcp 127.0.0.1:9090: connect: connection refused
+```
+
+NOTE: the `connection refused` message is expected. You will see the result in 5 minutes.
```

Comment:
When you say, "You will se the result in 5 minutes," do you mean `nettest` should take about 5 minutes to test connectivity? Being adjacent to `connection refused` makes this seem related.

Reply:
Clarification added in the above section `Run nettest`